src/filterConfig.js

```diff
@@ -23,3 +23,29 @@ export const browseInstanceIndexes = [
   { label: 'ui-inventory.browse.contributors', value: `${browseModeOptions.CONTRIBUTORS}`, queryTemplate: '%{query.query}' },
   { label: 'ui-inventory.browse.subjects', value: `${browseModeOptions.SUBJECTS}`, queryTemplate: '%{query.query}' },
 ];
+
+export const _browseInstanceIndexes = [
```

Comment:
Let's call this `newCallNumberBrowseInstanceIndexes` and add a comment to explain why we have 2 sets of indexes.